service/arm_templates/main.json

```diff
@@ -50,28 +50,81 @@
     "kubernetesIngressInsecurePort": {
       "type": "int"
     },
+    "masterCount": {
+      "type": "int",
+      "defaultValue": 1
+    },
+    "masterVMSize": {
+      "type": "string"
+    },
+    "masterDataDiskSizeGB": {
+      "type": "int"
+    },
+    "masterAdminUsername": {
+      "type": "string"
+    },
+    "masterDisablePasswordAuth": {
+      "type": "bool",
+      "defaultValue": true
+    },
+    "masterAdminSSHKeyData": {
+      "type": "string"
+    },
+    "masterOSImagePublisher": {
+      "type": "string"
+    },
+    "masterOSImageOffer": {
+      "type": "string"
+    },
+    "masterOSImageSKU": {
+      "type": "string"
+    },
+    "masterOSImageVersion": {
+      "type": "string"
+    },
+    "masterCloudConfigData": {
+      "type": "string",
+      "metadata": {
+        "description": "Base64-encoded cloud-config data."
+      }
+    },
     "templatesBaseURI" : {
       "type": "string",
       "metadata": {
         "description": "The base URI from which the templates will be fetched. Ends with a trailing '/'."
       }
     },
+    "availabilitySetsTemplateFile": {
+      "type": "string",
+      "defaultValue": "availability_sets_setup.json"
+    },
     "clusterSetupTemplateFile": {
       "type": "string",
       "defaultValue": "cluster_setup.json"
     },
+    "masterNodesSetupTemplateFile": {
+      "type": "string",
+      "defaultValue": "master_nodes_setup.json"
+    },
     "securityGroupsSetupTemplateFile": {
       "type": "string",
       "defaultValue": "security_groups_setup.json"
     },
+    "virtualMachinesSetupTemplateFile": {
+      "type": "string",
+      "defaultValue": "virtual_machines_setup.json"
+    },
```

Comment:
I don't see this file. Is it ninja fix?

Reply:
Good catch, forgot about that parameter that is unused. Just removed it from the file :)